src/components/analytics/ExternalLink.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { PropsWithChildren, ReactNode } from 'react'
+import { OutboundLink, OutboundLinkProps } from 'react-ga'
+
+type PropsOriginal = PropsWithChildren<OutboundLinkProps & React.HTMLProps<HTMLAnchorElement>>
+type Props = Omit<PropsOriginal, 'ref' | 'to' | 'eventLabel'> & {
+  eventLabel?: string
```

Comment:
I deliberately chosen to redefine this field by hand instead of using helpers like `Partial<Pick<OutboundLinkProps, 'eventLabel'>>`, I think it reads easier so I prefered to not make a TS helper hell

Same thing for the children

Reply:
You can make intermediary types to:
1. Avoid re-creating types
2. Increase readability

Like `PropsOriginal`

But I won't block you on this :)

Reply:
Yes, I can make a 2nd intermediate types. My point, is I feel is overkilling. I kind of find this setup a good balance

I just feel is easier to read `eventLabel?: string`, thant adding an intermediate `type with Partial<Pick<OutboundLinkProps, 'eventLabel'>>`